src/main/java/com/mixpanel/android/mpmetrics/MixpanelAPI.java

```diff
@@ -1577,9 +1733,10 @@ public InAppNotification getNotificationIfAvailable() {
 
         @Override
         public void trackNotificationSeen(InAppNotification notif) {
-            if(notif == null) return;
-
+            if (notif == null) return;
             mPersistentIdentity.saveCampaignAsSeen(notif.getId());
+            if (hasOptedOutTracking()) return;
```

Comment:
we need to be careful about this since decide uses `$campaign_delivery` to determine if a notification should be included in the decide response or not.